pkg/generate/app/imagestreamlookup.go

```diff
@@ -56,10 +58,20 @@ func (r ImageStreamSearcher) Search(terms ...string) (ComponentMatches, error) {
 				} else {
 					imageref, _ := imageapi.ParseDockerImageReference(term)
 					imageref.Name = stream.Name
+
+					localSearchTag := searchTag
```

Comment:
Clarity of intent - localSearchTag is not the same thing as a searchTag,
can be a better name though.

On Thu, Jan 21, 2016 at 11:45 AM, Ben Parees <notifications@github.com>
wrote:

> In pkg/generate/app/imagestreamlookup.go
> <https://github.com/openshift/origin/pull/6670#discussion_r50426932>:
>
> > @@ -56,10 +58,20 @@ func (r ImageStreamSearcher) Search(terms ...string) (ComponentMatches, error) {
> >  				} else {
> >  					imageref, _ := imageapi.ParseDockerImageReference(term)
> >  					imageref.Name = stream.Name
> > +
> > +					localSearchTag := searchTag
>
> why is this variable needed? the original searchTag value is never
> referenced again after you assign localSearchTag below, so you could have
> just continued using searchTag.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6670/files#r50426932>.
>